### PR TITLE
Issue/190

### PR DESCRIPTION
**Pull request by herau**

### CODE REVIEW

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java [hunk 1]
```diff
@@ -16,34 +16,11 @@
...
 import java.io.OutputStream;
-import java.util.Locale;
 
 public class Utils {
 
     public static final int MY_PERMISSIONS_REQUEST_CAMERA = 1;
     public static final int MY_PERMISSIONS_REQUEST_STORAGE= 2;
-    private static final String OPEN_FOOD_FACTS_WORLD_URL = "http://world.openfoodfacts.org/";
-    private static final String HTTP_FR_OPENFOODFACTS_ORG = "http://fr.openfoodfacts.org/";
-
```
Comment (teolemon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java]: Do you need a list of all locale with matching urls ?

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java [hunk 2]
```diff
@@ -16,34 +16,11 @@
...
-    public static String getUriByCurrentLanguage() {
-        String url;
-        if (Locale.getDefault().getLanguage().contains("fr")){
-            url = HTTP_FR_OPENFOODFACTS_ORG;
-        } else {
-            url = OPEN_FOOD_FACTS_WORLD_URL;
-        }
-        return url;
-    }
-
```
Comment (teolemon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java]: This shows the limits of the current system with countries like Switzerland (fr will default to France, while Swiss people also speak French…but also German and Italian :-D)

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java [hunk 3]
```diff
@@ -16,34 +16,11 @@
...
-    public static String getUriProductByCurrentLanguage() {
-        String url;
-        if (Locale.getDefault().getLanguage().contains("fr")){
-            url = HTTP_FR_OPENFOODFACTS_ORG + "produit/";
-        } else {
-            url = OPEN_FOOD_FACTS_WORLD_URL + "product/";
-        }
```
Comment (teolemon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java]: For the web version, we have put the "slugs": produit, contributeur… in separate strings.

Comment (teolemon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java]: fr => 'produit',
	de => 'produkt',
	en => 'product',
	el => 'προιον',
	es => 'producto',
	it => 'prodotto',
#	ru => 'продукт',
    ro => 'produs',
	ar => 'mountaj',
	pt => 'produto',
	he => 'mozar',
	nl => 'product',
	nl_be => 'product',

Comment (teolemon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java]: http://en.wiki.openfoodfacts.org/Translations_-_Web_site_interface

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java (hunk 2 above)
Comment (herau) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java]: that's why i remove this section. now all url are set in the string.xml files and each per language

**Pull request merged**